composer.json

```diff
@@ -50,7 +49,8 @@
     "justinrainbow/json-schema": "^2.0",
     "drupal/field_permissions": "~8.1.0-beta1",
     "drupal/address": "^8.1@RC",
-    "aws/aws-sdk-php": "~3.0"
+    "aws/aws-sdk-php": "~3.0",
+    "giggsey/libphonenumber-for-php": "^7.7"
```

Comment:
I will overlook this for now and perhaps it can be addressed in a future PR. I had a quick go and it didn't work.